courses-rest/src/main/java/no/sikt/nva/fs/Course.java

```diff
@@ -0,0 +1,63 @@
+package no.sikt.nva.fs;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+import nva.commons.core.JacocoGenerated;
+
+public final class Course {
```

Comment:
The objects should have a type.  Add  the annotation 
```@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")``` 
to the class

Reply:
It works as it is currently. Is it recommended to explicitly state it?

Reply:
Fixed